config/initializers/permissions.rb

```diff
@@ -177,6 +177,14 @@
                      permissible_on: :global,
                      require: :loggedin,
                      grant_to_admin: true
+
+      map.permission :manage_storages_in_project,
```

Comment:
🔴 This line is the one mentioned in daily.

It is preferrable IMHO, to keep storages related code inside the module code tree. So, we need a way to add those permissions from inside the storages module code. You said this is possible, @ba1ash ?

Reply:
Yes, that is possible. Have a look, pls.